app/scienceapi/events/models.py

```diff
@@ -70,15 +72,26 @@ class Event(models.Model):
         help_text='Slug appended to the Mozilla Science event\'s '
                   'url that represents this event',
     )
+    is_virtual = models.BooleanField(
+        default=False,
+        help_text='Check this box if your event is happening virtually, '
+                  'that is it\'s not tied to a particular location.'
+    )
+
+    ALL_TIMEZONES = [(i, i) for i in pytz.all_timezones]
+    timezone = models.CharField(
+        max_length=40,
+        choices=ALL_TIMEZONES,
+        help_text='Timezone of the place where the event will be held. '
+                  'If the event is virtual select any timezone you like. '
+                  'It is required that you input all datetimes in this '
+                  'timezone.'
+    )
+
     date_created = models.DateTimeField(auto_now_add=True)
     date_updated = models.DateTimeField(auto_now=True)
     starts_at = models.DateTimeField()
```

Comment:
9AM event in eastern standard will be stored as `14:00:00+00:00` as the time is in utc, when we convert it back to eastern standard the time will be `09:00:00-05:00`.
yes moment js can interpret that and we can send times like `09:00:00-05:00` to it.

I just thought all time related conversions should happen in one place